Readme.md

```diff
@@ -63,27 +63,58 @@ We provide SDKs for various programming languages to make it easy for you to int
-|Check file|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|
-|Check file list|&#9989;|&#9989;|&#10060;|&#9989;|&#9989;|&#10060;|&#10060;|&#9989;|
-|Custom Guids for tracability on user side|&#10060;|&#10060;|&#9989;|&#10060;|&#10060;|&#9989;|&#10060;|&#10060;|
+|Functionality|Rust|Java|PHP|TypeScript|.NET|Python|Ruby|Golang|C++|
+|---|---|---|---|---|---|---|---|---|---|
+|Check SHA256|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|
+|Check SHA256 list|&#9989;|&#9989;|&#10060;|&#9989;|&#9989;|&#10060;|&#10060;|&#9989;|&#10060;|
+|Check URL|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#10060;|
+|Check file|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|&#9989;|
+|Check file list|&#9989;|&#9989;|&#10060;|&#9989;|&#9989;|&#10060;|&#10060;|&#9989;|&#10060;|
+|Custom Guids for tracability on user side|&#10060;|&#10060;|&#9989;|&#10060;|&#10060;|&#9989;|&#10060;|&#10060;|&#10060;|
 
 
 ## Integration Ideas for Malware Detection trough VaaS
 You can use VaaS to create various applications that scan for malicious content with a few lines of code. Here are some examples:
 
-Create a command line scanner to find malware: [Example](rust/examples/gscan)
-<img src="assets/gscan.gif" alt="GScan command line malware scanner" style="width:100%">
+- [WordPress Plugin](https://wordpress.org/plugins/gdata-antivirus/) to scan for malware in uploaded files
+- [Nextcloud App](https://apps.nextcloud.com/apps/gdatavaas) to scan files in your Nextcloud instance
 
-Create a KDE Dolphin plugin to scan for malicious content: [Example](rust/examples/kde_dolphin)
-<img src="assets/dolphin_plugin.gif" alt="KDE Dolphin malware scanner plugin" style="width:100%">
+## Build & Test
+
+The easiest way to build and test the SDKs is with the [Nix Package Manager](https://nixos.org/download/#download-nix) and the provided [Just](https://github.com/casey/just) file. Nix will take care of all dependencies and Just provides a simple interface to run the most common tasks.
+
+To build and test the SDKs, run the following command:
+
+```bash
+# switch into a development shell with all dependencies installed.
+# This will not alter your system, but provide a shell with all necessary tools.
+nix develop
+
+# Now use the Just tool to run the most common tasks
+just -l # list all available tasks
+
+# Just Examples
+# Run the tests for the Rust SDK
+just test-rust
+```
+
+There are `test-*`, `build-*`, `clean-*` and `release-*` tasks for each SDK, with the exception of Python and PHP, where no build task is available. You can also run the tests for all SDKs with `just test-all`. A `build-all` and `clean-all` task is available as well.
+
+The `release-*` task triggers a Github Action to build and release a new version of the specified SDK. It needs a version number as an argument, which is used to tag the release. The version number should follow the [Semantic Versioning](https://semver.org/) scheme.
```

Comment:
does this actually create a github release?

Reply:
It's just a shortcut for `git tag -a v1.2.3 -m "Release ..." && git push origin v1.2.3`. That's how I released new versions of the SDKs. If a Github release is created is up to the CI pipeline, which I did not touch. So I'm not sure if the pipeline creates a release for the artifacts on push.

Reply:
Ah I see. We actually do it the other way around. Meaning we create a github-release in the UI and with the release we create the tag. We could discuss doing it your way and implement a github-release step into the pipelines.

Reply:
Makes sense. But for this PR I would not change anything as so far, both approaches seem to work fine.